README.rst

```diff
@@ -5,9 +5,9 @@ drizzle Documentation
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
 
-.. image:: https://travis-ci.org/spacetelescope/drizzle.svg?branch=master
-    :target: https://travis-ci.org/spacetelescope/drizzle
-    :alt: Travis CI
+.. image:: https://codecov.io/github/spacetelescope/drizzle/branch/master/graphs/badge.svg
```

Comment:
You'll also want to add the Github actions badge, in addition to the coverage one.

Reply:
I added this badge.